conf/report/report.css

```diff
@@ -3,17 +3,36 @@ body {
 }
 
 .headline {
-    text-align: center;
+  width: 100%;
+  text-align: center;
+  position: fixed;
+  display: table-row;
 }
 
 .headline a {  /* Only subtly hint that this is a link */
-    color: #00006f;
-    text-decoration: none;
+  color: #00006f;
+  text-decoration: none;
+}
+
+label {
+  position: fixed;
+  right: 0;
 }
 
 table.dataTable {
   width: auto;
   table-layout: fixed;
+  padding-top: 120px;
```

Comment:
I assume this was to make the text in the headers stop overlapping with the titles at the top - this is not a proper solution, this is what causes everything to flicker when starting to scroll down.